spec/api/v2/schemas/metadata.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    module Schemas
+      module Metadata
+        UUID = { type: :string, format: :uuid }.freeze
+        METADATA = {
```

Comment:
Logical separation, if you're doing it below, do it here as well:
```suggestion
        UUID = { type: :string, format: :uuid }.freeze
        
        METADATA = {
```